products/openssl.md

```diff
@@ -17,6 +17,12 @@ auto:
 
 # EOL dates and LTS infos on https://www.openssl.org/policies/releasestrat.html
 releases:
+-   releaseCycle: "3.3"
+    releaseDate: 2024-04-11
+    eol: 2026-04-11
+    latest: "3.3.0"
+    latestReleaseDate: 2024-04-11
```

Comment:
```suggestion
    releaseDate: 2024-04-10
    eol: 2026-04-10
    latest: "3.3.0"
    latestReleaseDate: 2024-04-10
```

According to https://www.openssl.org/blog/blog/2024/04/10/3.3-Final-Release/index.html.